src/main/flight/gps_rescue.c

```diff
@@ -394,19 +394,22 @@ static void performSanityChecks(void)
         previousDistanceToHomeCm = rescueState.sensor.distanceToHomeCm;
         secondsLowSats = 0;
         secondsDoingNothing = 0;
-        return;
```

Comment:
Did does the trick. Now sanity check is working again.

Reply:
@ctzsnooze this `return` blocked the sanity checks when `rescueState.phase` hangs on `RESCUE_INITIALIZE`